registry/server/common/services/validateRequest.ts

```diff
@@ -29,6 +29,7 @@ const validateRequestFactory = (validationConfig: ValidationConfig[]) => async (
         ));
         next();
     } catch (e) {
+        console.error(e);
```

Comment:
Why do we need it here? Don't think it will be a useful info in production env...